### PR TITLE
Add Dump UI World button and remap hotkeys

### PR DESCRIPTION
**Pull request by owenpalmer**
Fixes https://github.com/AmbientRun/Ambient/issues/193.

There was a bug that prevented `hotkey_modifier` from working, the modifiers weren't being set in the keyboard event.
I decided not to implement the write_debug function in this commit since I would rather not involve it in a minor feature such as this.

### CODE REVIEW

File: crates/app/src/lib.rs [hunk 1]
```diff
@@ -610,9 +609,7 @@ impl System<Event<'static, ()>> for ExamplesSystem {
                     },
                 ..
             } => match virtual_keycode {
```
Comment (FredrikNoren) [inline crates/app/src/lib.rs:611]: So these are actually used in the "native" examples, which doesn't have a Debugger. I'll merge and just revert that part

Comment (owenpalmer) [reply crates/app/src/lib.rs:611]: Okay that makes a lot more sense, thank you!

**Comment by FredrikNoren** [comment]
@owenpalmer Merged. Thanks for the PR!

**Comment by owenpalmer** [comment]
Happy to help!

**Pull request merged**